kubectl-plugin/component/component/alerting.go

```diff
@@ -0,0 +1,19 @@
+package component
+
+import "github.com/kubesphere-sigs/ks/kubectl-plugin/common"
+
+// Alerting return the struct of Alerting
+type Alerting struct {
+}
+
+// GetName return the name of Alerting
+func (e *Alerting) GetName() string {
+	return "alerting"
+}
+
+// Uninstall uninstall Alerting
+func (e *Alerting) Uninstall() error {
```

Comment:
Shall we implement a list  API to get the default CRs in this kubesphere built-in ns?   Therefore, we can directly remove these official CRs and not make affect the custom CRs.  For CRDs, we can add a flag to remove them or not.